assets/javascripts/discourse/templates/components/babble-icon.hbs

```diff
@@ -1,11 +1,13 @@
-{{#unless noTopicAvailable}}
-  {{#header-dropdown iconId="babble-notifications"
-                     icon="bullhorn"
-                     toggleVisible=babbleVisible
-                     loginAction="showLogin"
-                     title="babble.title"}}
-  {{/header-dropdown}}
-  {{#if unreadCount}}
-    <a href class='badge-notification unread-notifications'>{{unreadCount}}</a>
-  {{/if}}
-{{/unless}}
+{{#if Discourse.SiteSettings.babble_enabled}}
```

Comment:
I'd love to move this into a single sanity check with `noTopicAvailable`

What about
```
{{#if babbleEnabled}}
  ...
{{/if}}
```

```
Discourse.Babble.refresh = function(data) {
  self.set('babbleEnabled', Discourse.SiteSettings.babble_enabled && data.id
  if (!self.get('babbleEnabled') { return }
}
```

?